app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      def index
+        @books = Book.all
+        render_paginated json: @books
```

Comment:
You don't need to assign the books to a instance variable, you can use  a local variable or even  just `render_paginated Book.all`

Reply:
Thanks, Its done !